### PR TITLE
Firebase test

### PR DESCRIPTION
**Pull request by QuangTran304**
Guys. Have a look at the merge. I'm pretty sure that there will be conflicts.

### CODE REVIEW

**Review by QuangTran304 — COMMENTED** [review]
Everything looks good to me. I think it's safe to merge.

**Review by Klivree — APPROVED** [review]
I think it can be merged

**Comment by corinne-2000** [comment]
It looks good. It will be safe to merge.

**Review by anonna16 — APPROVED** [review]
Looks good and ready for merge!

File: src/Post.js [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
 
-
+  useEffect(() => {
+    database.collection('posts').orderBy("timestamp", "desc").onSnapshot(snapshot => {
+      setPosts(snapshot.docs.map( doc => ({
+        id: doc.id,
+        post: doc.data()
+      }) ));
+    })
+  }, []);
```
Comment (anonna16) [inline src/Post.js:32]: Pair programming: Anonna, Quang and Corinne
To link Post.js to the database (Firebase)

**Review by anonna16 — COMMENTED** [review]
Chnages in Post.js were made through pair programming

**Review by tusharraval102 — APPROVED** [review]
Looks good to me, sign in page and sign up page will change for next sprint

**Comment by shahdyousefak** [comment]
Looks good. Think its safe for merge

**Review by shahdyousefak — APPROVED** [review]
Looks good, can be safely merged

**Review by shahdyousefak — APPROVED** [review]
Reviewed & approved

**Review by corinne-2000 — APPROVED** [review]
The code seems to be correct. I believe could be merged onto main.

**Review by naregschoolwork — COMMENTED** [review]
Everything looks good, if any merge conflicts happen, we will fix it manually.

**Pull request merged**